documentation/src/main/asciidoc/topics/ref_server_realm_distributed.adoc

```diff
@@ -0,0 +1,14 @@
+[id='distributed_realm-{context}']
+= Distributed Realms
+A security realm definition for authentication and authorization identities distributed between multiple security realms.
```

Comment:
Can we add more context here. I have read this multiple times and I have no idea what this allows me todo as a Infinispan user.